### PR TITLE
Add hoverable menus and images

### PR DESCRIPTION
**Pull request by jabowering**
When an item from a menu is selected, a hoverable image and some
configurable text appear in the main display area.

### CODE REVIEW

File: css/frontPage.css [hunk 1]
```diff
@@ -143,3 +143,24 @@ footer {
     clear:             right;
     box-shadow:        4px 4px 2px rgba(0, 0, 0, 0.5);
 }
+
```
Comment (afmartin) [inline css/frontPage.css:146]: Could you remove the two extra blank lines and align the CSS property values in a column like the above CSS?

File: css/frontPage.css [hunk 2]
```diff
@@ -143,3 +143,24 @@ footer {
...
+    position: absolute;
+    left: 100%;
+    top: 10%;
+    background-color: #f9f9f9;
+    min-width: 160px;
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
+}
+
+.dropdown:hover .dropdown-content {
+    display: block;
```
Comment (afmartin) [inline css/frontPage.css:165]: The hover elements seem to appear under the original list.  Maybe try using the z-index property.

**Comment by afmartin** [comment]
Good work on the Javascript and drop down menus.  I just want to see the two small issues fixed before accepting this pull request.

**Comment by WatkinsEric** [comment]
I have a fix for the drop down menus going to merge and add it, along with clean up the code

**Pull request merged**